### PR TITLE
Fix nanite damage sensor and rule

### PR DESCRIPTION
**Pull request by WarlockD**
## About The Pull Request

Fix for issue [#48554](https://github.com/tgstation/tgstation/issues/48554)

## Why It's Good For The Game

It be nice for this to work.

## Changelog
:cl:
fix: Fixed nanite damage sensor and rule for sesnsor

/:cl:

### CODE REVIEW

**Comment by WarlockD** [comment]
Ugh I still have my old pull here.  Can it still be merged or do I have to get a clean fork first?  I am still new to git.

File: code/modules/research/nanites/nanite_programs/sensor.dm [hunk 1]
```diff
@@ -210,11 +210,12 @@
...
 			damage_amt = host_mob.getCloneLoss()
 
-	if(damage_amt >= damage.get_value())
-		if(check_above)
-			reached_threshold = TRUE
-	else if(!check_above)
-		reached_threshold = TRUE
+	if(check_above)
+		if(damage_amt >= damage.get_value())
+				reached_threshold = TRUE
```
Comment (Gamer025) [inline code/modules/research/nanites/nanite_programs/sensor.dm]: You have one tab too much here, remove one to fix indentation.
Thats why the tests are failing.

File: code/modules/research/nanites/nanite_programs/sensor.dm [hunk 2]
```diff
@@ -210,11 +210,12 @@
...
+	else
+		if(damage_amt < damage.get_value())
+				reached_threshold = TRUE
```
Comment (Gamer025) [inline code/modules/research/nanites/nanite_programs/sensor.dm]: Same as in line 215, remove one tab.

**Review by Gamer025 — COMMENTED** [review]
Tests are failing because wrong indentation.

File: code/modules/research/nanites/nanite_programs/sensor.dm (hunk 2 above)
Comment (WarlockD) [reply code/modules/research/nanites/nanite_programs/sensor.dm]: ugh sorry weird about the indentation.  Fixing now

**Review by WarlockD — COMMENTED** [review]
Changes

**Review by kriskog — APPROVED** [review]

**Comment by WarlockD** [comment]
Crap, I missed the get_value() on the rule and some reason it's still not proccing when installed as a program.  Do I reopen this or do another pull.

**Comment by kriskog** [comment]
Another PR please

**Pull request merged**